src/components/NotesMindMapView/NotesMindMapViewReducer.js

```diff
@@ -14,6 +16,8 @@ export const notesMindMapReducer = (state = defaultState, { type, data }) => {
       return { ...state, selectedNote: data };
     case CHANGE_NOTE_TEXT_ACTION:
       return { ...state, noteText: data };
+    case SET_CHANGE_PARENT_MODE_ACTION:
```

Comment:
Please conform to the Redux styleguide

>Model Actions as Events, Not Setters

https://redux.js.org/style-guide/style-guide#model-actions-as-events-not-setters